src/Orchard.Core.Tests/Orchard.Core.Tests.csproj

```diff
@@ -127,12 +127,12 @@
     <ProjectReference Include="..\Orchard.Web\Core\Orchard.Core.csproj">
       <Project>{9916839C-39FC-4CEB-A5AF-89CA7E87119F}</Project>
       <Name>Orchard.Core</Name>
-      <Private>false</Private>
+      <Private>True</Private>

```

Comment:
I needed to set this to true to be able to run the UT. Let me know if I shouldn't check this in